### DISCUSSION TITLE
List meetings by group instead of location

### DISCUSSION THREAD

**Discussion by dallasintergroup** [code4recovery/12-step-meeting-list#876] [Q&A]
Hi, thanks for this great plugin! It has been a HUGE help!

Several of our site users have expressed frustration at not being able to create a link which quickly pulls up a list of all meetings held by a group. The only option close to this currently, is to click on "...other meetings at this location" which isn't always helpful, since a location can host multiple groups. Additionally, some meetings of a group may be online, making the location different, so these meetings wouldn't appear on the location list. 

The only way I've found around this is to search for the desired meeting, switch results to "any day" and then copy the link for that search query. A clean and simple list of all meetings which are part of the same group would be a great benefit (was this not a feature before...I vaguely recall...?)

Thank you for your time and effort. 
Sam

**Comment by kiyote33** [comment]
Hi @dallasintergroup, thanks for the suggestion. I'll see that this gets discussed at our next team meeting.

**Comment by dallasintergroup** [comment]
Thank you Wayne!

*Sam Fuller*
Office Manager

*Dallas Intergroup Association*
4407 N Beltwood Pkwy
Suite 110
Farmers Branch TX 75244
214-887-6699
***@***.***
